.github/workflows/docs.yml

```diff
@@ -212,7 +212,7 @@ jobs:
 
       - name: Copy CI Results
         run: |
-          for board in p100 p100a p150a p300a; do
+          for board in p100a p150a p300a; do
```

Comment:
This should be changed to reference `.github/boards.json` as well.

Reply:
Yes, that would be good to go into another PR. @danieldegrasse - I didn't catch this in your PR earlier, unfortunately.

The main reason to reference the .json file is that we should avoid having arbitrary lists of boards scattered throughout our CI, because the lists do get desynchronized, and it's hit us in the past.